x/gov/keeper/abci.go

```diff
@@ -212,7 +218,15 @@ func (k Keeper) EndBlocker(ctx context.Context) error {
 				logMsg = "passed"
 
 				return nil
-			}); err != nil {
+			})
+			if err != nil {
+				// update proposal if error is out of gas
+				if errors.Is(err, sdkerrors.ErrOutOfGas) {
```

Comment:
should it be in all cases we set the proposal to fail? if not then we need to see how to make this work correctly

Reply:
Was this solved?

Reply:
Don't think so yet

Reply:
pushed what i think you mean, not sure where to store the proposal idx

Reply:
we need to remove L214-L219 and make the error log more generic (as it can be out of gas or proposal execution error (at handling)). As it is only used in the log, let's skip it?